.github/workflows/holonix-cache.yml

```diff
@@ -68,9 +68,9 @@ jobs:
       fail-fast: false
       matrix:
         extra_args:
-          - '--override-input versions ./versions/0_1'
-          - '--override-input versions ./versions/0_2'
-          - '--override-input versions ./versions/weekly'
+          - '--override-input versions "github:holochain/holochain?dir=versions/0_1"'
```

Comment:
The quotes are confusing this one - the value gets passed through as an environment variable so I'd just drop the quotes here and see if that behaves better. If that doesn't work I can take a look at the code